pkg/component/vpa/recommender.go

```diff
@@ -65,6 +65,8 @@ func (v *vpa) recommenderResourceConfigs() component.ResourceConfigs {
 		clusterRoleBindingMetricsReader   = v.emptyClusterRoleBinding("metrics-reader")
 		clusterRoleCheckpointActor        = v.emptyClusterRole("checkpoint-actor")
 		clusterRoleBindingCheckpointActor = v.emptyClusterRoleBinding("checkpoint-actor")
+		clusterRoleVPAStatusActor         = v.emptyClusterRole("vpa-status-actor")
```

Comment:
Minor nit: 
```suggestion
		clusterRoleVPAStatusActor         = v.emptyClusterRole("status-actor")
```
I see that for other resources we do not use the `vpa-` prefix as it will automatically be added to the name of the resource  like this `gardener.cloud:vpa:target:status-actor`

Reply:
done, thanks!